dimod/binary/binary_quadratic_model.py

```diff
@@ -1007,6 +1033,13 @@ def energy(self, sample, dtype: Optional[DTypeLike] = None) -> Bias:
         Returns:
             The energy.
 
+        Examples:
+            >>> bqm = dimod.BinaryQuadraticModel({}, {("x0", "x1"): -1}, "BINARY")
+            >>> bqm.energy(([1, 1], ["x0", "x1"]))
```

Comment:
IMO `{'x0': 1, 'x1': 1}` is clearer. I would leave this syntax for a discussion in `sample_like`.

Reply:
Okay, currently the examples show different formats, but they can show always the simplest

Reply:
I don't think we always need to show the simplest, but in this case I would consider the `(array_like, labels)` format to be a bit unintuitive, if a lot more performant.

Reply:
Agree, I updated